group-aws/boto3.md

```diff
@@ -0,0 +1,169 @@
+|----------------------|--------------------------------------------------------|
+| ec2_eni_facts        | [22941](https://github.com/ansible/ansible/pull/22941) |
+| ec2_group            | [25340](https://github.com/ansible/ansible/pull/25340) |
+| ec2_metric_alarm     | [23407](https://github.com/ansible/ansible/pull/23407) |
+| ec2_vpc_net_facts    | [25375](https://github.com/ansible/ansible/pull/25375) |
+| ec2_vpc_subnet_facts | [25374](https://github.com/ansible/ansible/pull/25374) |
+| rds                  | [25646](https://github.com/ansible/ansible/pull/25646) |
+| rds_param_group      | [25345](https://github.com/ansible/ansible/pull/25345) |
+| s3                   | [21529](https://github.com/ansible/ansible/pull/21529) |
+
+TODO: add many more to the above list!
+
+# State of the codebase (As at 2017-07-05)
+
+## boto3
+
+* `cloudformation_facts`
+* `cloudformation`
+* `cloudfront_facts`
+* `cloudtrail`
+* `ec2_ami_copy`
+* `ec2_asg`
+* `ec2_customer_gateway`
+* `ec2_eni_facts`
+* `ec2_instance_facts`
+* `ec2_lc_facts`
+* `ec2_snapshot_facts`
+* `ec2_vpc_igw_facts`
+* `ec2_vpc_nacl`
+* `ec2_vpc_peer`
+* `ec2_vpc_subnet`
+* `ec2_vpc_vgw_facts`
+* `ec2_vpc_vgw`
+* `ecs_cluster`
+* `ecs_ecr`
+* `ecs_service_facts`
+* `ecs_service`
+* `ecs_taskdefinition`
+* `ecs_task`
+* `efs_facts`
+* `efs`
+* `elasticache_parameter_group`
+* `elasticache_snapshot`
+* `elb_application_lb`
+* `elb_target_group_facts`
+* `elb_target_group`
+* `execute_lambda`
+* `iam_managed_policy`
+* `iam_mfa_device_facts`
+* `iam_server_certificate_facts`
+* `kinesis_stream`
+* `lambda_alias`
+* `lambda_event`
+* `lambda_facts`
+* `lambda`
+* `lightsail`
+* `route53_facts`
+* `s3_website`
+* `sts_session_token`
+
+## boto3 *and* boto
+
+* `eni_facts`
+
+## boto
+
+* `aws_api_gateway`
+* `dynamodb_table`
+* `ec2_ami_copy`
+* `ec2_ami_find`
+* `ec2_ami`
+* `ec2_eip`
+* `ec2_elb_facts`
+* `ec2_elb_lb`
+* `ec2_elb`
+* `ec2_eni_facts`
+* `ec2_eni`
+* `ec2_group`
+* `ec2_key`
+* `ec2_lc`
+* `ec2_metric_alarm`
+* `ec2`
+* `_ec2_remote_facts`
+* `ec2_scaling_policy`
+* `ec2_snapshot`
+* `ec2_tag`
+* `ec2_vol_facts`
+* `ec2_vol`
+* `ec2_vpc_dhcp_options`
+* `ec2_vpc_igw`
+* `ec2_vpc_net_facts`
+* `ec2_vpc_net`
+* `_ec2_vpc`
+* `ec2_vpc_route_table_facts`
+* `ec2_vpc_route_table`
+* `ec2_vpc_subnet_facts`
+* `ec2_win_password`
+* `ecs_cluster`
+* `ecs_service_facts`
+* `ecs_service`
+* `ecs_taskdefinition`
+* `ecs_task`
+* `elasticache`
+* `elasticache_subnet_group`
+* `iam_cert`
+* `iam_policy`
+* `iam`
+* `rds_param_group`
+* `rds`
+* `rds_subnet_group`
+* `redshift`
+* `redshift_subnet_group`
+* `route53_facts`
+* `route53_health_check`
+* `route53`
+* `route53_zone`
+* `s3_bucket`
+* `s3_lifecycle`
+* `s3_logging`
+* `s3`
+* `sns`
+* `sns_topic`
+* `sqs_queue`
+* `sts_assume_role`
+
+## Commands for the above list
+
+### boto3
+
+```
+grep -l 'import boto3' *.py | sed 's/\(.*\)\.py$/* `\1`/'
```

Comment:
Can we just grep for `HAS_BOTO` and `HAS_BOTO3`? I think all modules use those. This wouldn't catch `from boto3 import ....`